main/views.py

```diff
@@ -119,3 +121,22 @@ def get_context_data(self, **kwargs: Any) -> dict[str, Any]:  # type: ignore
         context = super().get_context_data(**kwargs)
         context["funding_name"] = str(self.get_object())
         return context
+
+
+class CapacityPlanningView(LoginRequiredMixin, TemplateView):
+    """View that renders the Capacity Planning page."""
+
+    template_name = "main/capacity_planning.html"
+
+    def get_context_data(self, **kwargs: Any) -> dict[str, Any]:  # type: ignore
+        """Add HTML components and Bokeh version to the context."""
+        context = super().get_context_data(**kwargs)
+        plot = plots.create_timeseries_plot(
+            datetime.now(), datetime.now() + timedelta(365)
+        )
+        script, div = components(plot)
+        context["script"] = script
+        context["div"] = div
+        version = plots.get_bokeh_version()
+        context["bokeh_version"] = version
```

Comment:
Why not just importing `bokeh` and getting the version directly? Don't see the need for a separate function doing this. 
```suggestion
        context["bokeh_version"] = bokeh.__version__
```